crates/bevy_reflect/src/struct_trait.rs

```diff
@@ -82,14 +154,19 @@ pub struct DynamicStruct {
 }
 
 impl DynamicStruct {
+    /// Returns the name of the struct.
```

Comment:
It may be good to say explicitly that this is the string-ified type of the represented struct (the same applies to the other `DynamicX::name()`)